cinema_service/settings.py

```diff
@@ -134,7 +139,7 @@
 STATIC_URL = "static/"
 
 MEDIA_URL = "/media/"
-MEDIA_ROOT = BASE_DIR / "media"
+MEDIA_ROOT = "/files/media"
```

Comment:
Code style issue: MEDIA_ROOT is typically set to os.path.join(BASE_DIR, 'media'). This ensures that the media folder is in your base directory. It's not a good practice to hardcode the path.